CognitiveSearch.UI/bin/Debug/netcoreapp2.0/CognitiveSearch.UI.runtimeconfig.dev.json

```diff
@@ -1,9 +1,8 @@
 {
   "runtimeOptions": {
     "additionalProbingPaths": [
-      "C:\\Users\\luisca\\.dotnet\\store\\|arch|\\|tfm|",
-      "C:\\Users\\luisca\\.nuget\\packages",
-      "C:\\Program Files (x86)\\Microsoft SDKs\\NuGetPackagesFallback",
+      "C:\\Users\\delegenz\\.dotnet\\store\\|arch|\\|tfm|",
+      "C:\\Users\\delegenz\\.nuget\\packages",

```

Comment:
We may want to delete this file too...

Reply:
I've removed this file in my most recent commit.